uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/ITestBroker.java

```diff
@@ -95,4 +95,10 @@
      * @return the {@link DeviceAdmin} name for this broker app
      */
     DeviceAdmin getAdminName();
+
+    /**
+     * Sets the flight information.
+     * @param flightsJson the json representation of the flight key and value pairs {"key1":"value"}.
+     */
+    void setFlights(@Nullable final String flightsJson);
```

Comment:
Should the param be `NonNull` as opposed to `Nullable`?

Reply:
I think it doesn't matter much, passing null or empty string will reset the flights.